app/uk/gov/hmrc/plasticpackagingtax/registration/views/registration_group.scala.html

```diff
@@ -74,7 +75,7 @@
         @task(4, "registrationPage.task.otherOrganisation.heading", Set(
             TaskSection(
                 title = "registrationPage.task.otherOrganisation",
-                link = pptRoutes.RegistrationController.displayPage(), //TODO
+                link = groupRoutes.OrganisationDetailsTypeController.displayPage(),
```

Comment:
OK for now.  But I think this link will be to my "group member list" controller eventually

Reply:
Yes.. I think then your controller would redirect if there are no members right??